app/helpers/application_helper.rb

```diff
@@ -13,8 +13,20 @@ def divide_in_chunks(arr, chunk_size)
     chunks
   end
 
-  def get_img_url(video)
-    video.img ? "/tmp/#{video.img}" : 'http://dummyimage.com/665x375/000000/00a2ff'
+  def get_large_cover_url(video)
```

Comment:
In case it wasn't clear, you would do that instead of having this helper. Either way works, It would come down to preference.